apb.yml

```diff
@@ -6,7 +6,7 @@ async: optional
 tags:
   - mobile-service
 metadata:
-  displayName: Mobile CI/CD
+  displayName: Mobile CI/CD (dev)
```

Comment:
why is it name (dev)?

Reply:
forgot to remove that, I use this name when I am testing the apb